PlaybackLab/MultiplePlayersScrollingViewSampleApp/MultiplePlayersScrollingViewSampleApp/Models/PlayerSelectionOption.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PlayerSelectionOption.swift
+//  MultiplePlayersScrollingViewSampleApp
+//
+//  Copyright © 2019 Ooyala. All rights reserved.
+//
+
+import UIKit
```

Comment:
```suggestion
class PlayerSelectionOption {
  
  public private(set) var domain: OOPlayerDomain
  
  public private(set) var pcode: String
  
  public private(set) var embedCode: String
  
  public private(set) var title: String
  
  public private(set) weak var embedTokenGenerator: OOEmbedTokenGenerator?

  init(pcode: String,
       embedCode: String,
       title: String,
       domain: OOPlayerDomain,
       embedTokenGenerator: OOEmbedTokenGenerator? = nil) {
    self.pcode = pcode
    self.embedCode = embedCode
    self.title = title
    self.domain = domain
    self.embedTokenGenerator = embedTokenGenerator
  }
}
```
we should do it in a Swift way

Reply:
Dpne

Reply:
I can't see your changes